src/renderer/components/+add-cluster/add-cluster.tsx

```diff
@@ -374,9 +379,12 @@ export class AddCluster extends React.Component {
         <div className="actions-panel">
           <Button
             primary
+            disabled={addDisabled}
             label={<Trans>Add cluster(s)</Trans>}
             onClick={this.addClusters}
             waiting={this.isWaiting}
+            tooltip={addDisabled ? "Select at least one cluster to add." : undefined}
```

Comment:
should this be localized? hmm, looks like we have other tooltips not localized.

Also, might be clearer to say 'cluster context' since the drop-down says "Select contexts". Actually I think the drop-down should say "Select cluster contexts"

Reply:
Probably should be localized anyway